Jenkinsfile

```diff
@@ -1,109 +1,109 @@
+//            contents = readFile 'version.txt'
+//            def version = contents.trim()
+//            def artifactoryPort
+//            if ("${BUILD_TYPE}" == "releases") {
+//              artifactoryPort = '8446'
+//            } else {
+//              artifactoryPort = '8445'
+//            }
+//            def dockerImage = docker.build(DOCKER_IMAGE_NAME)
+//            sh "docker tag ${DOCKER_IMAGE_NAME} ${artifactoryHost}:${artifactoryPort}/${ARTIFACTORY_PATH}/${DOCKER_IMAGE_NAME}:${version}"
+//            sh "docker tag ${DOCKER_IMAGE_NAME} ${gitlabHost}:5001/${GITLAB_PATH}/${DOCKER_IMAGE_NAME}:${version}"
+//            sh "docker tag ${DOCKER_IMAGE_NAME} ${ecrHost}/${DOCKER_IMAGE_NAME}:${version}"
+//            sh "docker tag ${DOCKER_IMAGE_NAME} ${ecrHost}/${DOCKER_IMAGE_NAME}:latest"
+//            sh "docker tag ${DOCKER_IMAGE_NAME} usgswma/${DOCKER_IMAGE_NAME}:${version}"
+//          }
+//        }
+//      }
+//    }
+//    stage('Publish Docker images') {
+//      steps {
+//        withCredentials([
+//          string(credentialsId: 'ARTIFACTORY_HOST', variable: 'artifactoryHost'),
+//          string(credentialsId: 'GITLAB_HOST', variable: 'gitlabHost'),
+//          string(credentialsId: 'ECR_HOST', variable: 'ecrHost')
+//          ]) {
+//          script {
+//            contents = readFile 'version.txt'
+//            def version = contents.trim()
+//            if ("${BUILD_TYPE}" == "releases") {
+//              artifactoryPort = '8446'
+//            } else {
+//              artifactoryPort = '8445'
+//            }
+//            // Push to Artifactory
+//            withDockerRegistry([credentialsId: 'ARTIFACTORY_UPLOADER_CREDENTIALS', url: "https://${artifactoryHost}:${artifactoryPort}"]) {
+//              sh "docker push ${artifactoryHost}:${artifactoryPort}/${ARTIFACTORY_PATH}/${DOCKER_IMAGE_NAME}:${version}"
+//            }
+//            // Push to Gitlab
+//            withDockerRegistry([credentialsId: 'jenkins_ci_access_token', url: 'https://${gitlabHost}:5001']) {
+//              sh "docker push ${gitlabHost}:5001/${GITLAB_PATH}/${DOCKER_IMAGE_NAME}:${version}"
+//            }
+//            // Push to ECR
+//            sh "#!/bin/sh -e\n" + "eval \$(aws --region us-west-2 ecr get-login --no-include-email)"
+//            sh "docker push ${ecrHost}/${DOCKER_IMAGE_NAME}:${version}"
+//            sh "docker push ${ecrHost}/${DOCKER_IMAGE_NAME}:latest"
+//          }
+//        }
+//      }
+//    }
+//    stage('Push bumped version to git') {
+//      steps {
+//        withCredentials([string(credentialsId: 'GITHUB_ACCESS_TOKEN', variable: 'jenkinsToken')]) {
+//          sh '''
+//            if [ $BUILD_TYPE == "releases" ]
+//            then
+//              $WORKSPACE/env/bin/bumpversion --allow-dirty --no-tag minor
+//              /usr/bin/git push
+//              /usr/bin/git push --tags
+//            fi
+//          '''
+//        }
+//      }
+//    }
+//  }
+//}
```

Comment:
Any reason Jenkinsfile was kept with all lines commented? Seems like it could be deleted.

Reply:
I commented it out because I thought it could be removed but wasn't sure.  :) 

If I check in and everything works, I'll delete it.